docs/user-guide/monitoring.md

```diff
@@ -1,11 +1,15 @@
 # Prometheus and Grafana installation
+Two different methods to install and configure Prometheus and Grafana are described in this doc. 
+- Prometheus and Grafana installation using Pod Annotations. This installs Prometheus and Grafana in the same namespace as NGINX Ingress
+- Prometheus and Grafana installation using Service Monitors. This is more suited for existing Prometheus and Grafana installations that can be configured with NGINX Ingress for scraping the metrics. 
```

Comment:
Maybe also explain which one is preferred method? For a newcomer it is not possible to evaluate that by them self without rigorous googling. Prometheus maintainers apparently them self would prefer ServiceMonitors: https://github.com/prometheus-operator/prometheus-operator/issues/1547#issuecomment-401092041

That's probably also why prometheus helm charts does not support annotation based method out of the box.

Reply:
To me understanding this took hours of investigation for example. Plenty of documentation that I found still uses pod annotations as an example, which might have been the preferred method before.

Reply:
@indhupriya are you going to implement the suggestions from @villesau? We can also leave that open for a follow-up task in order to get this one done.

Reply:
Grafana dashboard by following the doc in this pr looks empty. There is no data. Not sure if its the Grafana or Service monitor that isn't configured properly. Im still investigating that.

Reply:
Which grafana dashboard are you referring to? Try to install this dashboard to see nginx metrics: https://grafana.com/grafana/dashboards/9614 However, it probably remains empty during your testing if there's no traffic trough your ingress during testing.

Reply:
That makes sense @villesau I guess there's no traffic.

Reply:
ok, so back to the question if the PR is finished for now or if you want to implement the suggestions from @villesau

Reply:
@iamNoah1 Yeah, I can include changes suggested by @villesau. @longwuyuan also suggested a few changes. I'll create a PR with all those.